.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,6 +4,8 @@ about: Report a problem
 
 ---
 
+<!-- We'll respond to big bugs, but probably not smaller ones. -->
```

Comment:
Inspired by ACF's great support topic: https://wordpress.org/support/topic/free-support-expectations-and-response-times-7/